src/layout/Tabs/config.ts

```diff
@@ -0,0 +1,61 @@
+import { CG, Variant } from 'src/codegen/CG';
+import { CompCategory } from 'src/layout/common';
+
+export const Config = new CG.component({
+  category: CompCategory.Container,
+  rendersWithLabel: false,
+  capabilities: {
+    renderInTable: false,
+    renderInButtonGroup: false,
+    renderInAccordion: false,
+    renderInAccordionGroup: false,
+    renderInTabs: false,
+    renderInCards: false,
+    renderInCardsMedia: false,
+  },
+})
+  .addTextResource(
+    new CG.trb({
+      name: 'ariaLabel',
+      title: 'Aria Label',
+      description: 'The aria label for the tabs',
+    }),
+  )
+  .addProperty(new CG.prop('size', new CG.enum('small', 'medium', 'large').optional()))
```

Comment:
In `.optional()` you can also specify what's the default setting if none are provided. :raised_hands: